hayanime

```diff
@@ -339,11 +339,12 @@ hist_update(){
 	curl -sLfk "${base_url}/v1/spy-x-family" -A "${user_agent}" > /dev/null 2>&1 || die 1 "\033[2K\rServer can't be reach..."
 	while read -r entry; do
 		: "$((entriter += 1))"
-		cur_ep="${entry#*€}" cur_ep="${cur_ep%%£*}" cur_ep="${cur_ep%%/*}"
+		cur_ep="${entry#*€}" cur_ep="${cur_ep%%£*}" last_tot_ep="${cur_ep##*/}" cur_ep="${cur_ep%%/*}"
 		tot_ep="$(curl -sLk "${base_url}${entry%%¢*}" -A "${user_agent}" | sed -nE 's_.*eptotal":([^\,\}]*)(,.*|\}</.*)_\1_p')"
 		printf '%s\n' "${entry}" | sed "s%\(.*€\)\(.*\)\(£.*\)%\\1${cur_ep}/${tot_ep}\\3%" >> ~/"${hist_file}".tmp
+		[ "${debug_mode}" = "true" ] && [ -n "${tot_ep}" ] && [ "${last_tot_ep}" -lt "${tot_ep}" ] && ani_updated="${entry##"${entry%%¢*}"¢}" ani_updated="${ani_updated%%€*}" && debug_handler ani-updated "${ani_updated}" "$(( tot_ep - last_tot_ep ))" "${tot_ep}"
 		progbar "${entriter}" "${hist_count}" "Updating..."
-		unset cur_ep
+		unset cur_ep tot_ep last_tot_ep
```

Comment:
This will look smthn like this
![Screenshot_20220903_193629](https://user-images.githubusercontent.com/93582751/188268286-1b9d3df6-93ee-4104-9b28-ae2f981b7516.JPG)

Reply:
Just like listing for new releases of an anime in history. looks cool